tests/functional/pv/pv_services/test_rwop_pvc.py

```diff
@@ -72,6 +73,63 @@ def test_pod_with_same_priority(self, pod_factory, interface):
 
         self.create_pod_and_validate_pending(pod_factory, interface)
 
+    def test_pvc_clone_and_snapshot(
```

Comment:
is it intentionaly that we don't have "tier" mark set? it will not be triggered during regression